src/System.Xml.XmlSerializer/src/System/Collections/ArrayList.cs

```diff
@@ -17,7 +16,6 @@
 namespace System.Collections
 {
     [DebuggerDisplay("Count = {Count}")]
-    [System.Runtime.InteropServices.ComVisible(true)]
     internal class ArrayList : List<object>
```

Comment:
@khdang Any ideas why we have a duplicated ArrayList as opposed to using something like LowLeveList or `List<T>`.